client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -2,11 +2,13 @@ import { ThemeProvider } from '@emotion/react';
 import styled from '@emotion/styled';
 import { useTranslate } from 'i18n-calypso';
 import { useEffect, useState } from 'react';
+import { useIsFetching } from 'react-query';
```

Comment:
It's nice to see `useIsFetching` be used 👍

Reply:
❤️